opentracing/harness/api_check.py

```diff
@@ -40,6 +40,14 @@ def check_baggage_values(self):
         """
         return True
 
+    def check_trace_identifiers(self):
+        """If true, the test will validate trace ID and :class:`Span` ID by
+        retrieving them from the trace context. If false, it will only attempt
+        to retrieve them to check the API compliance, but not actually validate
+        the values. The latter mode is only useful for no-op tracer.
```

Comment:
I am not sure what the difference between true and false. How can you **validate** the returned values? The actual content of those values is not dictated by the spec.

Reply:
ah, nvm, I see what it checks.